src/Forms/Fieldset.tsx

```diff
@@ -59,6 +60,9 @@ const StyledFieldset = styled.fieldset<StyledFieldsetProps>`
   padding-inline-end: 0.75em;
   padding-block-end: 0.625em;
   min-inline-size: min-content;
+  & > * {
+    margin: ${fromTheme('ws', 'small')};
```

Comment:
I don't necessarily know if there's a right answerto this, but with the margin added to the child component do you think we still need the padding inside the `FieldSet` as well? 

I've looked at it both ways a couple times and I _think_ I prefer it without the padding on the top and bottom of the fieldset, but I think it might make since to keep the padding on the left and right. What do you think?

Reply:
I think that we can remove the padding on the top and bottom too.

However, I noticed one thing with the margin which was that this new margin rule was making it so that the checkbox, which needs to be in a fieldset for accessibility reasons, is indented.
<img width="314" alt="Screen Shot 2021-02-11 at 4 10 27 PM" src="https://user-images.githubusercontent.com/29589689/107701696-404b6600-6c87-11eb-8030-c90f098a74ac.png">
 I updated the margin rule to only add padding to the `margin-top` and `margin-bottom`. What do you think?

Reply:
Ah, now that you point that out it's super noticable. Good eye :eye: 

I think these latest changes look great, and that combination of margins and paddings should be all set.